docs/src/main/asciidoc/spring-cloud-bus-pubsub.adoc

```diff
@@ -0,0 +1,36 @@
+== Spring Cloud Bus
+
+Spring Cloud Bus integration with GCP Pub/Sub is as simple as importing two Spring Cloud dependencies: `spring-cloud-bus` and `spring-cloud-gcp-pubsub-stream-binder`.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-bus</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-gcp-pubsub-stream-binder</artifactId>
+    </dependency>
+----
+
+Gradle coordinates:
+
+[source,subs="normal"]
```

Comment:
Doesn't look like we need `subs` in this block: there is just no any placeholders.
But adding `groovy` instead would be great!